src/services/logger.ts

```diff
@@ -72,14 +72,23 @@ export function debugObj(e: any) {
 // everywhere else `throw` the error upstream
 // TODO is there a way to DRY that?
 // returns the sentry error id
-export function logUserErr(e: any): string {
+export function logUserErr(e: any, dbId?: string, uid?: string): string {
   let err = e?.message ?? '';
   let errStack = err;
   if (e.metadata?.failures) {
     err = e.metadata.failures.map((f: Error) => f?.message).join('\n');
     errStack = e.metadata.failures.map((f: Error) => f?.stack ?? f?.message).join('\n');
   }
-  if (config.sentry) err += `\nPlease provide the following error ID if reporting it to the IaSQL team: ${sentry.captureException(errStack)}`;
+  if (config.sentry) {
+    err += `\nPlease provide the following error ID if reporting it to the IaSQL team: ${sentry.captureException(errStack, {
+      // the id or the username need to be present
+      // https://docs.sentry.io/platforms/node/guides/express/enriching-events/identify-user/
+      user: {
+        id: uid,
+        username: dbId
```

Comment:
The 'username' seems weird?

Reply:
just going off the cleanest way to bucket events to a user or database. the scheduler doesn't have access to the uid so dubbing the dbId as the username, in amplitude we do something similar with the device id